amazonia-1/amazonia-1-tiler-example.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Comment or Text block describing that this will request and RGB image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>